spyder/utils/tests/test_programs.py

```diff
@@ -98,8 +98,9 @@ def test_run_python_script_in_terminal(scriptpath, qtbot):
 @flaky(max_runs=3)
 @pytest.mark.first
 @pytest.mark.skipif(
-    os.environ.get('CI', None) is None or os.name == 'nt',
-    reason='Only on CI and not on windows!',
+    (os.environ.get('CI', None) is None
+     or (sys.platform.startswith("linux") or os.name == "nt")),
+    reason='Only on CI and only mac!',
```

Comment:
Please revert this. It was caused by not installing a terminal on the Linux CIs. But I solved that several weeks ago.